packages/perseus-editor/src/widgets/expression-editor.tsx

```diff
@@ -282,7 +283,7 @@ class ExpressionEditor extends React.Component<any, any> {
         if (this.props.answerForms.length === 0) {
             issues.push("No answers specified");
         } else {
-            const hasCorrect = !!_(this.props.answerForms).find((form) => {
+            const hasCorrect = !!this.props.answerForms.find((form) => {
```

Comment:
This could use `.some()` instead of `.find()` and get rid of the `!!` prefix trick to convert the numeric value to a boolean.